.github/workflows/e2e.yml

```diff
@@ -17,12 +21,17 @@ jobs:
         run: |
           npm install
           npm run package
-      - run: |
-          npx playwright test
+      - name: E2E test on win
+        run: npx playwright test
+        if: ${{ matrix.os != 'ubuntu-latest' }}
```

Comment:
better to be `matrix.os == 'windows-latest'`?

Reply:
maybe later we could support macos, but there still some problem, so not onboard the macos.
will investigate later